app/models.py

```diff
@@ -96,6 +96,8 @@ async def average(db, query):
                 measurements.c.latitude,
                 measurements.c.longitude,
                 func.avg(measurements.c.pm2dot5).label("average"),
+                func.max(measurements.c.pm2dot5).label("maximum"),
+                func.min(measurements.c.pm2dot5).label("minimum"),
```

Comment:
the name of this function should be change to `def stats`, because now returns more than just the average.

Reply:
@sofiab97 please add a commit fixing this before our review on Friday ;)